File: DotaMaster/DotaMaster.Data/Entities/Item.cs
namespace DotaMaster.Data.Entities
{
    public class Item
    {
        public int Id { get; set; }
        public required string Name { get; set; }
        public required string DisplayName { get; set; }
        public required string Lore { get; set; }
        public int? Cost { get; set; }
        public List<Attribute>? Attributes { get; set; }
        public required string IconUrl { get; set; }
        public required string Description { get; set; }
    }
    public class Attribute
    {
        public required string Name { get; set; }
        public required string Value { get; set; }
    }
}

File: dota-master-client/src/shared/models/item.ts
export default interface Item {
  id: number;
  name: string;
  displayName: string;
  lore: string;
  cost?: number;
  iconUrl: string;
  description: string;
  attributes: Attribute[];
}

interface Attribute {
  name: string;
  value: string;
}
